test/suite/functional/Generator/ClientFactory/ClientFactoryDefault.php

```diff
@@ -18,25 +16,17 @@
 class SwaggerPetstoreClientFactory
 {
     /**
-     * @param string $baseUri
-     * @param array  $options
+     * If using Guzzle 6, make sure to configure Guzzle to not throw exceptions
```

Comment:
Let's put it to README instead, because it would be annoying to see this comment if you don't use Guzzle6(which you should not, because it is outdated).

Reply:
done! README is created with it on the example, so anybody using the generated client will be aware of it